make/migrations/postgresql/0160_2.13.0_schema.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE p2p_preheat_policy DROP COLUMN IF EXISTS scope;
```

Comment:
It is better to keep the old column even if it is not used anymore, because it still keep the previous configuration to reference.

Reply:
I think we do not need to retain this column, as the new design is using `extra_attrs` to store it.